AppServer/google/appengine/api/taskqueue/taskqueue.py

```diff
@@ -864,17 +865,17 @@ def __host_from_target(target):
     """
     default_hostname = app_identity.get_default_version_hostname()
     if default_hostname is None:
-
-
-
       return None
+    # AppScale: We do not support the instance parameter.
+    target_info = target.split('.')
+    if len(target_info) > 2:
+      raise InvalidTaskError(
```

Comment:
Yes, you're right. When I wrote the comment, I was under the impression that the add request had a target field.